server/v2/testdata/app.toml

```diff
@@ -21,6 +21,7 @@ mock_field_two = 1
 minimum-gas-prices = '0stake'
 
 [store]
+Home = ''
```

Comment:
??

Reply:
hmmm not sure how this happened, I didn't tag the field.  maybe it needs an explicit ignore tag?